frontends/github-pages/src/App.tsx

```diff
@@ -16,7 +16,7 @@ const PageTitle = styled.h1`
   margin-top: 6rem;
   margin-bottom: 0.5rem;
   font-size: 50px;
-  color: ${({ theme }) => theme.custom.colorBlue5};
+  color: "#03152d";
```

Comment:
This was the only usage. it's not one of the design system colors: https://www.figma.com/file/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=0%3A1&mode=dev Maybe I ought have done it in a separate PR, but I was a bit aggressive in cleaning up the old color variables in this PR. My thought is:

**Is the color in the [color designs](https://www.figma.com/file/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=0%3A1&mode=dev)**?
- Yes: then it goes in Theme
- No. Then:
    - it will be replaced anyway when that page gets proper designs
    - Replace it with something close that is in the theme for now
    - or, failing that, leave it hardcoded as raw value.  

Probably I should have just replaced this with black, the color the designs are using for generic text.